src/interfaces.ts

```diff
@@ -316,3 +316,46 @@ export interface ISdkOptions {
 export interface IFeatureFlags {
     [key: string]: number | boolean | string;
 }
+
+export type GuidType = 'guid';
+export type RGBType = 'rgb';
+
+export interface IRGBColor {
```

Comment:
there is already `IColor` interface which has the same attributes

Reply:
Good catch, thanks!